base/mqueue/platform_event.go

```diff
@@ -13,38 +13,31 @@ type PlatformEvent struct {
 	ID          string                  `json:"id"`
 	Type        *string                 `json:"type"`
 	Timestamp   *types.Rfc3339Timestamp `json:"timestamp"`
-	Account     *string                 `json:"account"`
 	AccountID   int                     `json:"account_id"`
```

Comment:
What is AccountID and how does it play with OrgID?

Reply:
I'm not completely sure, but based on the code `AccountID` is not actually coming from inventory but it is used internally for queries using `rh_account_id` column

Reply:
I see, it's internal ID. We should probably name it better.